scss/_base_fontfaces.scss

```diff
@@ -1,106 +1,224 @@
 @mixin vf-b-typography-fontfaces {
   @if str-index($font-base-family, 'Ubuntu') {
-    @font-face {
-      font-family: 'Ubuntu';
-      font-style: normal;
-      font-weight: 300;
-      src: url('#{$assets-path}e8c07df6-Ubuntu-L_W.woff2') format('woff2'), url('#{$assets-path}8619add2-Ubuntu-L_W.woff') format('woff');
-    }
+    @if $font-use-subset-latin {
+      @font-face {
+        @if $font-display-option {
+          font-display: $font-display-option;
+        }
```

Comment:
This is a lot of duplication that need to be maintained if there is a change. You could include all the common styles in a mixin and include it. For example, https://www.sassmeister.com/gist/839dc206787ad1545d3db1d9744cbb9b

Reply:
or is the default of `auto` a cleaner option?

Reply:
I moved to the auto option, made that the default and updated the docs.  Also removed the preload on examples as it was giving a console warning for being after the actual resource was loaded, which I think is good.